go/vt/sqlparser/comments.go

```diff
@@ -396,3 +397,49 @@ func Consolidator(stmt Statement) querypb.ExecuteOptions_Consolidator {
 	}
 	return querypb.ExecuteOptions_CONSOLIDATOR_UNSPECIFIED
 }
+
+// GetWorkloadFromComments returns the value of the workload as specified in the query comments. It assumes the presence
+// of a comment that includes a string in the form `workloadLabel=workloadname;` (including the semicolon). The function
+// returns `unspecified` if it fails to obtain the workload from the query comments. If there are multiple comments that
+// specify the workload, or multiple workload specifications in the same comment, it returns the first one. It trims
+// whitespaces both at the beginning and end of the workload name.
+func GetWorkloadFromComments(query string, workloadLabel string) string {
```

Comment:
I do not think we need this method. Can just use `IsSet` and `GetString` methods already available on `CommentDirectives`

Reply:
Will use that, thanks.

Reply:
Got rid of this. Replaced with another method to extract the specific workload directive from the `Statement`, to emulate what is done in other places in the code.